amundsen_application/static/js/utils/googleAnalytics.ts

```diff
@@ -0,0 +1,22 @@
+import ua from 'universal-analytics';
```

Comment:
This was a quick and not studied integration, happy to use a more lightweight that only wraps the event sending code of Google Analytics.